react-hot-button-component/my-app/src/HotButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+type Props = {
+  color: string;
+};
+
+export default function HotButton({ color }: Props) {
+  const [count, setCount] = useState(0);
+
+  function handleClick() {
+    setCount(count + 3);
```

Comment:
We want to change the button based on how many times a user clicked it, so it should be `setCount(count + 1)`!